academic-services/knowledge-exploration-service/tutorial-conference-filterable-paperlist.md

```diff
@@ -0,0 +1,382 @@
+    {"name": "VenueShortName","type": "blob?"}
+  ]
+}
+```
+
+### Filter attributes
+
+Filter attributes are attributes that can be used to filter entity data.
+
+The numeric attributes that we want to filter by would be the conference paper's **Year, CitationCount, EstimatedCitationCount**. Depending on the filter UX we want to provide, we can add **equals** and/or **is_between** operations to the filterable numeric attributes.
+
+>[!NOTE]
+>Try adding **is_between** to numeric filter attributes and extend the sample code to enables publication year range filter.
+
+The following schema elements reflects the filterable numeric attributes:
+
+```json
+{
+  "attributes": [
+    {"name": "Id","type": "int64!","operations": [ "equals" ]},
+    {"name": "Year","type": "int32?","operations": [ "equals", "is_between" ]},
+    {"name": "CitationCount","type": "int32?","operations": [ "equals", "is_between" ]},
+    {"name": "EstimatedCitationCount","type": "int32?","operations": [ "equals", "is_between" ]},
+    {"name": "AuthorAffiliations.Sequence","type": "int32?","operations": [ "equals" ]}
+  ]
+}
+```
+
+For string attributes, we want to select attributes that common values such as journal name, conference name. For attribute values that may be too noisy, you may opt for the normalized version such as using **AuthorName** instead of **OriginalAuthorName**. we should add **equals** operation to these attributes.
+
+The following schema elements reflects the filterable string attributes:
+
+```json
+{
+  "attributes": [
+    {"name": "DOI","type": "string?","operations": [ "equals", "starts_with" ]},
+    {"name": "Title","type": "string?","operations": [ "equals", "starts_with" ]},
+    {"name": "TitleWords","type": "string*","operations": [ "equals" ]},
+    {"name": "AbstractWords","type": "string*", "operations": [ "equals" ]},
+  
+    {"name": "AuthorAffiliations","type": "Composite*"},
+    {"name": "AuthorAffiliations.AuthorName","type": "string?","operations": [ "equals", "starts_with" ]},
+    {"name": "AuthorAffiliations.AffiliationName","type": "string?","operations": [ "equals", "starts_with" ]},
+
+    {"name": "FieldsOfStudy","type": "Composite*"},
+    {"name": "FieldsOfStudy.Name","type": "string?","operations": [ "equals", "starts_with" ]},
+
+    {"name": "Conference","type": "Composite?"},
+    {"name": "Conference.Name","type": "string?","operations": [ "equals", "starts_with" ]},
+  ]
+}
+```
+
+We've include a complete schema for you to compare against. See **<tutorial_resource_root>/kddSchema.josn** for the complete schema.
+
+## Build a custom paper index
+
+Once you're ready with your schema. We can start building a MAKES index for the KDD data.
+
+Since the index we're building is relatively small and simple, we can build this locally on a dev machine. If the index you're building is large or contains more complex operations, use cloud index build to leverage high performing machines in Azure.
```

Comment:
>Azure. [](start = 244, length = 6)

To learn more, follow [How to create index from MAG](how-to-create-index-from-mag.md)